package.json

```diff
@@ -19,27 +19,28 @@
     "victory": "^0.22.1"
   },
   "devDependencies": {
-    "@storybook/addon-actions": "^3.4.6",
-    "@storybook/addon-links": "^3.4.6",
-    "@storybook/addon-storysource": "^3.4.6",
-    "@storybook/addons": "^3.4.6",
-    "@storybook/react": "^3.4.6",
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.3",
+    "@babel/cli": "^7.2.3",
+    "@babel/core": "^7.3.4",
+    "@babel/preset-env": "^7.3.4",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/runtime": "^7.3.4",
+    "@storybook/addon-actions": "^5.0.1",
+    "@storybook/addon-links": "^5.0.1",
+    "@storybook/addon-storysource": "^5.0.1",
+    "@storybook/addons": "^5.0.1",
+    "@storybook/react": "^5.0.1",
+    "babel-loader": "^8.0.5",
     "babel-plugin-inline-react-svg": "^0.4.0",
     "babel-plugin-styled-components": "^1.5.0",
-    "babel-preset-es2015": "^6.24.1",
-    "babel-preset-react": "^6.24.1",
-    "babel-runtime": "^6.26.0",
     "cheerio": "^1.0.0-rc.2",
-    "enzyme": "^3.6.0",
-    "enzyme-adapter-react-16": "^1.5.0",
-    "enzyme-to-json": "^3.3.4",
+    "enzyme": "^3.9.0",
+    "enzyme-adapter-react-16": "1.5.0",
+    "enzyme-to-json": "^3.3.5",
     "eslint": "^4.19.1",
     "eslint-plugin-react": "^7.7.0",
-    "jest": "^23.6.0",
-    "jest-styled-components": "^6.2.0",
-    "react-test-renderer": "^16.5.2"
+    "jest": "^24.3.1",
+    "jest-styled-components": "^6.3.1",
```

Comment:
@sarathms let's not forget to file a ticket about jest-styled-components and it blocking on us upgrading to styled-components v4.

Reply:
Filed in #49